scripts/prod.sh

```
@@ -0,0 +1,148 @@
+        print_message "error" "Failed to start docker containers"
+        exit 1
+    fi
+}
+
+# Main function to run all tasks
+main() {
+    build_images
+    start_containers
+
+    echo -e "\n\n"
+    if [ $? -eq 0 ]; then
```

Shellcheck (0.10.0)

[warning] This $? refers to echo/printf, not a previous command. Assign to variable to avoid it being overwritten.

(SC2320)